pkgs/development/python-modules/steam/default.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  cachetools,
+  gevent,
+  gevent-eventemitter,
+  protobuf,
+  pycryptodomex,
+  requests,
+  six,
+  vdf,
+  win-inet-pton,
+}:
+buildPythonPackage rec {
+  pname = "steam";
+  version = "1.4.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ValvePython";
+    repo = "steam";
+    rev = "v${version}";
+    hash = "sha256-OY04GsX3KMPvpsQl8sUurzFyJu+JKpES8B0iD6Z5uyw=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    cachetools
+    gevent
+    gevent-eventemitter
+    protobuf
+    pycryptodomex
+    requests
+    six
+    vdf
+    win-inet-pton
+  ];
+
```

Comment:
Require the tests a internet connection?

Reply:
most did not, seems i forgot to go back and check which deps had tests